supervisor/misc/tasks.py

```diff
@@ -147,20 +150,13 @@ async def _update_addons(self):
             except AddonsError:
                 _LOGGER.error("Can't auto update Add-on %s", addon.slug)
 
+    @Condition.free_space
+    @Condition.healthy
```

Comment:
Let's simplify it: `@Job(condition=[JobCondition.FREESPACE, JobCondition.HEALTHY, JobCondition.INTERNET])`